lavalink/client.py

```diff
@@ -60,7 +60,8 @@ def add_event_hook(self, hook):
         if hook not in self._event_hooks:
             self._event_hooks.append(hook)
 
-    def add_node(self, host: str, port: int, password: str, region: str, name: str = None):
+    def add_node(self, host: str, port: int, password: str, region: str,
+                 resume_key: str = None, resume_timeout: int = None, name: str = None):
```

Comment:
How about setting the default `resume_timeout` to 60 seconds ? (according to [lavalink implementation.md](https://github.com/Frederikam/Lavalink/blob/master/IMPLEMENTATION.md#resuming-lavalink-sessions)) ?